app/javascript/dashboard/components-next/NewConversation/helpers/composeConversationHelper.js

```diff
@@ -193,10 +198,12 @@ export const searchContacts = async ({ keys, query }) => {
   return filteredPayload || [];
 };
 
-export const createNewContact = async email => {
+export const createNewContact = async input => {
   const payload = {
-    name: getCapitalizedNameFromEmail(email),
-    email,
+    name: input.startsWith('+')
```

Comment:
Better to check using regex, something like this `[+0-9\s()]+`

Reply:
We had a discussion on this, so I’m resolving this comment.